src/azure-cli/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -139,6 +142,38 @@ def create_cluster(cmd, client, cluster_name, resource_group_name, cluster_type,
         raise CLIError('Either the kafka client group id and kafka client group name should be specified, '
                        'or none of them should be')
 
+    # Validate and initialize autoscale setting
+    autoscale_configuration = None
+    if autoscale_type == 'Load':
```

Comment:
>'Load' [](start = 25, length = 6)

1. consider define variables/enums for constant strings?
2. Is this string comparsion case insensitive?

Reply:
fixed.

---
In reply to: [466333934](https://github.com/Azure/azure-cli/pull/14692#discussion_r466333934) [](ancestors = 466333934)